client/src/app/components/stardust/block/BlockTangleState.tsx

```diff
@@ -1,19 +1,20 @@
 import classNames from "classnames";
-import moment from "moment";
 import React, { useEffect, useState } from "react";
 import { BlockTangleStateProps } from "./BlockTangleStateProps";
 import { useMilestoneDetails } from "~helpers/hooks/useMilestoneDetails";
 import Tooltip from "../../Tooltip";
 import "./BlockTangleState.scss";
+import { DateHelper } from "~/helpers/dateHelper";
 
 const BlockTangleState: React.FC<BlockTangleStateProps> = ({ network, status, milestoneIndex, hasConflicts, conflictReason, onClick }) => {
-    const [ago, setAgo] = useState<string | undefined>();
+    const [milestoneTimestamp, setMilestoneTimestamp] = useState<string | undefined>();
     const [blockId, setBlockId] = useState<string | undefined>();
     const [milestoneDetails] = useMilestoneDetails(network, milestoneIndex ?? null);
 
     useEffect(() => {
         if (milestoneDetails?.milestone?.timestamp) {
-            setAgo(moment(milestoneDetails.milestone?.timestamp * 1000).fromNow());
+            const readableTimestamp = DateHelper.format(DateHelper.milliseconds(milestoneDetails.milestone.timestamp));
```

Comment:
I would add a hyphen next to the date so it looks a bit better
![image](https://github.com/iotaledger/explorer/assets/3624944/042d348b-4f37-40b3-8862-d2b17e366fff)